### ISSUE TITLE
[A11Y] Discussion reply count (mark as read) is button with no accessible information

### ISSUE THREAD

**Issue by davwheat** [flarum/framework#3368]
![image](https://user-images.githubusercontent.com/7406822/126401258-dea5204d-f28a-4357-84e6-5e5e71566aa8.png)

While having these as buttons make sense to visual users, using ChromeVox allows us to hear what a screenreader user will hear:

> test general flarumite replied 8 days ago
>
> 1 button

Preferably this should be "1 reply". Multiple replies should be "3 replies". The unread version should be "1 new reply. Mark as read.".

**Comment by SychO9** [comment]
fixed in https://github.com/flarum/framework/pull/3090
